src/xdocs/google_style.xml

```diff
@@ -21,10 +21,10 @@
                 <p>
                     This coverage report was created for
                     <a href="http://google.github.io/styleguide/javaguide.html">
-                        Google Java Style</a>(
-                    <a href="styleguides/google-java-style-20170228.html">
-                          cached page</a>)
-                    , version of 28 February 2017
+                        Google Java Style</a>(<a
+                        href="styleguides/google-java-style-20180523/google.github.io/styleguide/javaguide.html"
+                    >cached page</a>),
+                    version of 23 May 2018, current as of 30 March 2019
```

Comment:
I am not sure why we need "30 March", what is reason to have it ?
Not sure why user need to know date of export. We use date as some sort of anchor to commit in googe style repo.
it is probably better to place  SHA1 of commit/link-to-commit to be even more exact.

Reply:
Well, I as user would be interested in this information tbh.
It is more visible with the sun style where we have "version of 20 April 1999, current as of 30 March 2019".
It shows the user when was last checked that the cached and reported-on version is current.
If I only read "version of 20 April 1999" or "version of 23 May 2018", I ask myself "is this also compatible with the latest version, were there any changes?".
If I see that it was last checked on "30 March 2019", I'm pretty confident the report is against a pretty recent version of the styleguide.
(Updated to 07 May 2019 now, as I just rechecked it is still recent)